src/css/custom.scss

```
@@ -556,7 +556,7 @@ body .DocSearch {
     var(--ifm-color-emphasis-100) 100%
   );
 }
-// External link icon indicator.
+// External link icon indicator (excluded from navbar).
```

Stylelint (17.12.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)